packages/x-charts/src/models/axis.ts

```diff
@@ -61,15 +61,32 @@ export interface ChartsAxisProps extends TickParams {
   /**
    * The font size of the axis ticks text.
    * @default 12
+   * @deprecated Consider using  `tickLabelStyle.fontSize` instead.
```

Comment:
Nit: Useless empty space.
```suggestion
   * @deprecated Consider using `tickLabelStyle.fontSize` instead.
```